composer.json

```diff
@@ -75,7 +75,7 @@
     },
     "require-dev": {
         "aws/aws-sdk-php": "~3.0",
-        "iron-io/iron_mq": "~2.0",
+        "iron-io/iron_mq": "~2.0 || ~4.0",
```

Comment:
please use `~2.0|~4.0`, and also update the other 2 places where this occurs too